doc/classes/ProjectSettings.xml

```diff
@@ -733,6 +733,10 @@
 		</member>
 		<member name="input/ui_text_scroll_up.OSX" type="Dictionary" setter="" getter="">
 		</member>
+		<member name="input/ui_text_search_next" type="Dictionary" setter="" getter="">
+		</member>
+		<member name="input/ui_text_search_previous" type="Dictionary" setter="" getter="">
+		</member>
```

Comment:
How come these are added to the doco? I can't find references to them anywhere else.

Reply:
These shortcuts were added in a previous commit. I renamed them to address feedback, but forgot to run the doctool.